File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        result=[]
        for i in nums:
            if val!=i:
                result.append(i)
        for i in range(len(result)): 
               nums[i]=result[i]
        return len(result)       
                

File: 0027-remove-element/0027-remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int n=nums.length;
        int[] result=new int[n];
        int count=0;
        for(int i : nums){
            if (val!=i){
                result[count]=i;
                count++;
            }
        }
        for(int i=0;i<count;i++){
            nums[i]=result[i];
        }
        return count;
    }
}